Marlin/src/feature/bedlevel/ubl/ubl_G29.cpp

```diff
@@ -1252,7 +1252,7 @@
                 // last half of the mesh (when every unprobed mesh point is one index
                 // from a probed location).
 
-                d1 = HYPOT(i - k, j - l) + (1.0 / ((millis() % 47) + 13));
+                d1 = HYPOT(i - k, j - l) + RECIPROCAL((millis() % 47) + 13);
```

Comment:
`RECIPROCAL` is safer, but also a little slower. If we know the parameter won't ever be 0.0 then it's better to just use `1.0 / x`.

Reply:
You are right here. That is why a review is always welcome !! 👍